Packs/ApacheTomcat/ModelingRules/ApacheTomcatModelingRules_1_3/ApacheTomcatModelingRules_1_3_testdata.json

```diff
@@ -0,0 +1,22 @@
+{
+    "data": [
+        {
+            "test_data_event_id": "b71645d8-853e-498f-8545-f60f3085cbea",
```

Comment:
Can this (and all other id's) be changed to a value that's clearly bogus?

Reply:
I believe only if it's a valid UUID - so any sequence where there are 8-4-4-4-12 alphanumeric characters.